tests/ui/chalkify/assert.rs

```diff
@@ -1,6 +0,0 @@
-// run-pass
```

Comment:
lol where did these tests go

Reply:
Fixing them would have required implementing `Callable` in chalk